### PR TITLE
Enable conditional directives

### PR DESCRIPTION
**Pull request by collinjackson**

### CODE REVIEW

**Comment by Hixie** [comment]
LGTM, cc @johnmccutchan

File: sky/engine/core/script/dart_init.cc [hunk 1]
```diff
@@ -71,6 +71,7 @@ static const char* kDartProfilingArgs[] = {
     // default profile period to 100Hz. This number is suitable for older
     // Raspberry Pi devices but quite low for current smartphones.
     "--profile_period=1000",
+    "--conditional_directives",
```
Comment (chinmaygarde) [inline sky/engine/core/script/dart_init.cc]: This ends up in `kDartProfilingArgs`. Not sure conditional_directives are related to profiling.

Comment (collinjackson) [inline sky/engine/core/script/dart_init.cc]: Thanks, they're not. I've moved them.

**Comment by johnmccutchan** [comment]
LGTM

**Pull request merged**